plugins/vpc-bridge/config/k8s/connector_binary.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+//go:build enablek8sconnector && windows
+
+package k8s
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+
+	log "github.com/cihub/seelog"
+)
+
+const (
+	// Represents Env variable for k8s connector binary path.
+	envK8sConnectorBinaryPath = "AWS_VPC_CNI_K8S_CONNECTOR_BINARY_PATH"
+	// Represents default path for k8s connector binary in EKS Windows AMIs.
+	defaultK8sConnectorBinaryPath = `C:\Program Files\Amazon\EKS\cni\aws-vpc-cni-k8s-connector.exe`
+
+	// Represents Env for log level for k8s connector binary.
+	envLogLevel = "VPC_CNI_LOG_LEVEL"
+	// Represents default log level for k8s connector binary.
+	defaultLogLevel = "info"
+)
+
+// connectorBinary Represents k8s connector binary.
+type connectorBinary struct {
+	binaryPath, binaryLogLevel string
+}
+
+// newConnectorBinary returns connector binary.
+func newConnectorBinary() connectorBinary {
```

Comment:
there is no benefit to having a struct here. Please just do this as part of the execute command path. Much simpler to read and understand

Reply:
done